cranelift-codegen/meta/src/shared/instructions.rs

```diff
@@ -1606,10 +1604,15 @@ pub(crate) fn define(
         sge    uge      Greater than or equal
         sgt    ugt      Greater than
         sle    ule      Less than or equal
+        of     *        Overflow
+        nof    *        No Overflow
         ====== ======== =========
 
         When this instruction compares integer vectors, it returns a boolean
         vector of lane-wise comparisons.
+
+        * The unsigned version of overflow conditions have ISA-specific
+        semantics and thus have been kept as methods on the TargetIsa trait.
```

Comment:
Since this ends up in doc comments, would it be useful to put the name and/or a link to the unsigned_add_overflow_condition function?